### ISSUE TITLE
API to turn off video/audio

### ISSUE THREAD

**Issue by kno10** [thecodingmachine/workadventure#1443]
E.g. when linking to a zoom room, it would be great to be able to turn off the camera and microphone in WA when the user clicks the "open zoom" button. An API for this would be good to have.

**Comment by moufmouf** [comment]
True! Adding this issue to the list of improvements, thanks for the idea!

**Comment by kno10** [comment]
I tried using silent zones as a workaround, but these appear to only disable the "circles" functionality, not to turn off camera and microphone.
